botocore/__init__.py

```diff
@@ -51,6 +51,9 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+# Used to specify anonymous (unsigned) request signature
+UNSIGNED = ''
```

Comment:
That is a backward-incompatible change for the signature version events fired from the `RequestSigner`, which is why I kept the empty string.